pupil_src/shared_modules/calibration_routines/finish_calibration.py

```diff
@@ -10,14 +10,18 @@
 """
 
 import os
-import numpy as np
 
-from . import calibrate
```

Comment:
Isn't `calibrate` used further down? e.g. https://github.com/pupil-labs/pupil/pull/1367/files#diff-6325cc3346b4eb436ce02ce0aa3a38aaR348

Reply:
In the meantime, I have renamed it to calibrate_2d and import the only function needed in finish_calibration explicitly.